### PR TITLE
Laser Calibration Table

### PR DESCRIPTION
**Pull request by fabiansinz**

### CODE REVIEW

File: python/pipeline/experiment.py [hunk 1]
```diff
@@ -170,6 +172,60 @@ class Software(dj.Lookup):
         ('imager', '1.0')]
 
 @schema
+class LaserCalibration(dj.Manual):
+    definition = """
+    # stores measured values from the laser power calibration
+
+    calibration_id      : int
+    ---
+    -> Person
```
Comment (dimitri-yatsenko) [inline python/pipeline/experiment.py]: I would add an explicit `calibration_date` attribute for those cases when the calibration date differs from the time that the data are entered in the database.

Comment (dimitri-yatsenko) [inline python/pipeline/experiment.py]: Is it unreasonable to use a natural primary key for `LaserCalibration` e.g. (`->Rig, `calibration_date`)

Comment (dimitri-yatsenko) [inline python/pipeline/experiment.py]: Why is software part of this table?

Comment (fabiansinz) [inline python/pipeline/experiment.py]: Because it handles all the calibration that could influence the measurement (e.g. the Pockels cell).

**Comment by fabiansinz** [comment]
I'll change the other things. Wait with acceptance.

**Pull request merged**